test/Microsoft.AspNetCore.SignalR.Client.Tests/TestConnection.cs

```diff
@@ -17,7 +18,7 @@
 
 namespace Microsoft.AspNetCore.SignalR.Client.Tests
 {
-    internal class TestConnection : IConnection
+    internal class TestConnection : ConnectionContext, IConnection
```

Comment:
Why is `IConnection` still around?

Reply:
We pass it to the ITransport. I plan to get rid of it when I merge this https://github.com/aspnet/SignalR/pull/1893

Reply:
Am removing it now